app/models/ability.rb

```diff
@@ -0,0 +1,14 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    if user.role == 'admin'
```

Comment:
- [ ] [ OPTIONAL] Instead of direcly using `user.role == 'admin'` her ,you can define a function in user model like.
```
def admin?
     role == 'admin' ? true : false
   end
```
 & call it here by user.admin?, because in the future you can directly check admin by this.